enhancements/template.md

```diff
@@ -150,15 +150,20 @@ determine graduation.
 
 Consider the following in developing the graduation criteria for this
 enhancement:
-- Maturity levels - `Dev Preview`, `Tech Preview`, `GA`
-- Deprecation
+- [Maturity levels (`alpha`, `beta`, `stable`)][maturity-levels]
```

Comment:
we don't have alpha, beta, stable.  We have dev preview, tech preview, and GA.

Reply:
> we don't have alpha, beta, stable...

Even if we don't add any, we still supply upstream Kubernetes features which are alpha/beta/stable.

> We have dev preview, tech preview, and GA

Do we have docs for that?  I looked (references in the topic comment), and found nothing on dev-preview and little on tect-preview.

Reply:
There's another PR to document tech preview guidelines -- https://github.com/openshift/enhancements/pull/104